integration/init/just-ship-yaml/metadata.yaml

```
@@ -0,0 +1,3 @@
+upstream: "https://github.com/replicatedhq/test-charts/tree/ebebc9e692db3caeadb308ddeec37e9565acba1e/just-ship-yaml"
+args: []
```

```suggestion
args: ["--prefer-git"]
```
Since that doesn't hit GitHub rate limits which makes it easier to run the integration tests locally